### PR TITLE
Feature/tr 1821/keayboard navigation in the text reader pci

### PR DESCRIPTION
**Pull request by krampstudio**
Related to https://oat-sa.atlassian.net/browse/TR-1821

TextReader PCI (IMS version) 
 - Add keyboard navigation for the tabs 
 - Add focus style for the tabs  and the navigation buttons
 - Update the test playground
 - Migration script

How to test:
 - You can test in the **item preview** (create an item with a TextReader PCI)
 - You can test locally by launching the test playground `cd tao/views/build && npx grunt connect:test:keepalive` and open the interaction test with your web browser
 - From the version 3.1.0 of the extension, switch to this branch and execute the following migration `sudo -u www-data php index.php '\oat\tao\scripts\tools\Migrations' -c migrate -v 'oat\pciSamples\migrations\Version202110280904504106_pciSamples' -e pciSamples`

> Please note the Terre test runner (and test preview) prevents tab navigation inside items. 
> Please note a bug in the Solar test runner backend prevents to launch PCIS. 

Testing env:
 - [Kitchen Playground](http://test-tr-1821-textreader.playground.kitchen.it.taocloud.org:41021/tao/Main/login)
 - [Credentials](http://playground.kitchen.it.taocloud.org/?p=test-tr-1821-textreader_delivery.git;a=blob;f=app/admin.txt;h=caff2176c2325ef1004d7c503f9a8778e3e55c8a;hb=refs/heads/master)

### CODE REVIEW

File: migrations/Version202110280904504106_pciSamples.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            )(
+                ['1.1.0']
+            )
+        );
+
+    }
+
+    public function down(Schema $schema): void
+    {
+         $this->addReport(
```
Comment (meefox) [inline migrations/Version202110280904504106_pciSamples.php:36]: Everything will be ok, bundles, css, etc?
I noticed most of the migrations are throwing IrreversibleMigration exeption

Comment (krampstudio) [reply migrations/Version202110280904504106_pciSamples.php:36]: @quintanilhar do you mind to check if I'm following the best practices for the migration scripts, thanks

Comment (krampstudio) [reply migrations/Version202110280904504106_pciSamples.php:36]: JS confirmed me we need the irreversible exception, I'll fix it.

**Review by meefox — APPROVED** [review]
Looking good!
Screenshot 2021-11-09 at 13 50 42


#
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by marpesia — APPROVED** [review]
LGTM, checked only on TAE env.

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x]  Commits messages are meaningful
- [ ]  Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: migrations/Version202110280904504106_pciSamples.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202110280904504106_pciSamples extends AbstractMigration
+{
+
+    public function getDescription(): string
+    {
+        return 'Add basic keayboard navigation to the TextReaderInteraction';
```
Comment (quintanilhar) [inline migrations/Version202110280904504106_pciSamples.php:21]: Small typo: `keayboard `
```suggestion
        return 'Add basic keyboard navigation to the TextReaderInteraction';
```

**Review by quintanilhar — APPROVED** [review]

**Pull request merged**